spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/core/QueueOperation.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.core;
+
+/**
+ * Operations for queue service.
```

Comment:
It's too simple and ambiguous.  
1. What operation do you really try to provide?
2. What's queue service?